core-web/apps/dotcms-ui/src/app/view/components/dot-toolbar/components/dot-login-as/dot-login-as.component.html

```diff
@@ -1,10 +1,15 @@
-<dot-dialog
-    (hide)="close()"
+<p-dialog
     [(visible)]="visible"
     [header]="'login-as' | dm"
-    [actions]="dialogActions">
+    [modal]="true"
+    [draggable]="false"
+    [resizable]="false"
+    [style]="{ width: '400px' }"
+    [baseZIndex]="10000"
```

Comment:
think baseZIndex being 10000 is a hard-coded solution. Is there any other solution?